Gems/Multiplayer/Code/Source/Components/LocalPredictionPlayerInputComponent.cpp

```diff
@@ -391,7 +395,7 @@ namespace Multiplayer
         {
             // Reprocess the input for this frame
             NetworkInput& input = m_inputHistory[replayIndex];
-            ScopedAlterTime scopedTime(input.GetHostFrameId(), input.GetHostTimeMs(), invokingConnection->GetConnectionId());
+            ScopedAlterTime scopedTime(input.GetHostFrameId(), input.GetHostTimeMs(), DefaultBlendFactor, invokingConnection->GetConnectionId());
```

Comment:
Same, we want the same blending logic for time that you have above here too. Even though this is on a client correct, when we replay these inputs if one of those inputs relied on some kind of blended state (think two moving players running into each other for example), we need that blended state to be correct on input replay as well.

Reply:
That seems reasonable, I'll need to check that input on the client actually has blend factor populated.